quarkus/service/src/test/java/org/apache/polaris/service/quarkus/admin/PolarisOverlappingTableTest.java

```diff
@@ -79,15 +79,15 @@ static Stream<Arguments> testTableLocationRestrictions() {
             "ALLOW_UNSTRUCTURED_TABLE_LOCATION", "false", "ALLOW_TABLE_LOCATION_OVERLAP", "false");
     Map<String, Object> laxCatalog =
         Map.of(
-            ALLOW_UNSTRUCTURED_TABLE_LOCATION.catalogConfig(),
+            ALLOW_UNSTRUCTURED_TABLE_LOCATION.catalogConfigs().getFirst(),
             "true",
-            ALLOW_TABLE_LOCATION_OVERLAP.catalogConfig(),
+            ALLOW_TABLE_LOCATION_OVERLAP.catalogConfigs().getFirst(),
```

Comment:
optional suggestion: `.catalogConfigs().getFirst()` -> `.canonicalName()` (or just keep the old `catalogConfig()` with updated javadoc.

Reply:
I think you're right to point out how awkward `catalogConfigs().getFirst()` is, but in the current implementation it's not really clear which catalog config is meant to be the "canonical name". There's a deeper problem here, let me try to adjust the way we track the safe / unsafe configs.